opm/simulators/flow/BlackoilModelParametersEbos.hpp

```diff
@@ -432,6 +440,7 @@ namespace Opm
             EWOMS_REGISTER_PARAM(TypeTag, int, StrictInnerIterMsWells, "Number of inner iterations for multi-segment wells with strict tolerance");
             EWOMS_REGISTER_PARAM(TypeTag, bool, UseInnerIterationsWells, "Use nested iterations for standard wells");
             EWOMS_REGISTER_PARAM(TypeTag, int, MaxInnerIterWells, "Maximum number of inner iterations for standard wells");
+            EWOMS_REGISTER_PARAM(TypeTag, bool, AlternativeWellRateInit, "Use alternative well rate initialization procedure");
             EWOMS_REGISTER_PARAM(TypeTag, Scalar, RegularizationFactorMsw, "Regularization factor for ms wells");
```

Comment:
Since you don't want this to be default. When does the alternative well rate scaling fail? Currently it is used at the beginning of an time step could it be an idea to only use it for new wells.

Reply:
I have not seen it fail. But since it is different, it causes all the tests to need data updates, which means we have to test and verify. Keeping the original default separates that issue from the network option work.